src/app/components/WithLoader/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ActivityIndicator, View } from 'react-native';
+import { CERULEAN } from '@constants/colors';
+
+import styles from './styles';
+
+interface Props {
+  loading: boolean;
+  children: JSX.Element;
+}
+
+function WithLoader(props: Props) {
```

Comment:
Generally, we destructure the `props` object (`{ loading, children }: Props`) but it's ok